### PR TITLE
Fixed calling deprecated setDefaultOptions method

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is a patch.

## Changelog

```markdown
### Fixed
 - Fixed calling deprecated setDefaultOptions method
```

## Subject

This is patch is needed for symfony 3.

### CODE REVIEW

**Comment by OskarStark** [comment]
@core23 please take care of adding the labels according to your PR, thank you 🍻

File: src/Form/Type/CommentType.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ public function getName()
     /**
      * {@inheritdoc}
      */
-    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    public function configureOptions(OptionsResolver $resolver)
```
Comment (jordisala1991) [inline src/Form/Type/CommentType.php]: Isn't this BC break? 

Maybe mantain both method like we did on all other bundles?

Comment (greg0ire) [reply src/Form/Type/CommentType.php]: I think it's acceptable. Bear in mind that `setDefaultOptions` isn't called anymore now that we dropped sf < 2.8

Comment (core23) [reply src/Form/Type/CommentType.php]: This is handled by symfony itself: https://github.com/symfony/symfony/blob/v2.8.0/src/Symfony/Component/Form/AbstractType.php#L47

Comment (jordisala1991) [reply src/Form/Type/CommentType.php]: okay, then I dont know why we did it different on all the other cases.

Comment (greg0ire) [reply src/Form/Type/CommentType.php]: Maybe it was before dropping sf < 2.8?

Comment (jordisala1991) [reply src/Form/Type/CommentType.php]: Could be indeed

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**